static/js/templates.js

```diff
@@ -69,7 +69,9 @@ Handlebars.registerHelper('tr', function (context, options) {
     //     1. `context` is very important. It can be `this` or an
     //        object or key of the current context.
     //     2. Use `__` instead of `{{` and `}}` to declare expressions
-    const result = i18n.t(options.fn(context), context);
+    let context_string = options.fn(context);
+    context_string=context_string.replace(/\n/g," ").replace(/    /g,'').trim();
```

Comment:
Would it work to instead trim all leading spaces, rather than all groups of 4 spaces?  This seems like it might mangle strings that have 4 spaces in them somewhere else.

Reply:
The code we need to be consistent with is in
[`zerver.management.commands.makemessages.Command.extract_strings`](https://github.com/zulip/zulip/blob/2c4101dbc5ea376dcba368f4fa3033151e67c5c3/zerver/management/commands/makemessages.py#L143-L152).

Reply:
Trimming all leading spaces would work, I'm pushing the commit with the change.